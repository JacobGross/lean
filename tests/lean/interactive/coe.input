VISIT coe.lean
WAIT
INFO 5 16