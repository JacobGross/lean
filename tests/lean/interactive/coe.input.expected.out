-- BEGINWAIT
-- ENDWAIT
-- BEGININFO
-- TYPE|5|16
decidable (a = b)
-- ACK
-- SYNTH|5|16
has_decidable_eq a b
-- ACK
-- SYMBOL|5|16
_
-- ACK
-- ENDINFO
